code/game/objects/items/toys.dm

```diff
@@ -830,12 +825,7 @@
 			qdel(C)
 		else
 			user << "<span class='warning'>You can't mix cards from other decks!</span>"
-		if(cards.len > 26)
-			src.icon_state = "deck_[deckstyle]_full"
-		else if(cards.len > 10)
-			src.icon_state = "deck_[deckstyle]_half"
-		else if(cards.len > 1)
-			src.icon_state = "deck_[deckstyle]_low"
+	update_icon()
 
 /obj/item/toy/cards/deck/MouseDrop(atom/over_object)
```

Comment:
unrelated to this pr but if i remember correctly there's no way to make a deck out of normal cards and i think it's ridicolous,any chance you can fix that?